frontend/app/src/store/session/auth.ts

```diff
@@ -44,6 +49,38 @@ export const useSessionAuthStore = defineStore('session/auth', () => {
     set(dbUpgradeStatus, null);
   };
 
+  const getUserAccounts = async () => {
+    const accounts = await usersApi.getUsers();
+    set(userAccounts, accounts);
+  };
+
+  const checkIfLogged = async (username: string): Promise<boolean> => {
+    if (!get(userAccounts)) {
+      await getUserAccounts();
+    }
+    const accounts = get(userAccounts) ?? {};
+    return accounts[username] === LOGGED_IN;
+  };
+
+  const loggedUsers = async (): Promise<string[]> => {
+    if (!get(userAccounts)) {
+      await getUserAccounts();
+    }
+    const loggedUsers: string[] = [];
+    const accounts = get(userAccounts) ?? {};
+    for (const user in accounts) {
+      if (accounts[user] !== LOGGED_IN) {
+        continue;
+      }
+      loggedUsers.push(user);
+    }
+    return loggedUsers;
+  };
+
+  const users: ComputedRef<string[]> = computed(() =>
+    Object.keys(get(userAccounts) ?? {})
+  );
+
```

Comment:
Is it really necessary to store the accounts in the pinia store?
Other than avoiding duplicate requests maybe during login?

The information is relevant only while you are at the login screen/login in.

Reply:
I was trying to avoid multiple calls, but, yes, I can revert to use multiple calls over storing in pinia.